io_scene_nif/modules/animation/animation_import.py

```diff
@@ -467,26 +478,29 @@ def import_kf_standalone(self, kf_root, b_armature_obj, bind_data):
                 kfc = controlledblock.controller
                 # fallout, Loki
                 if not kfc:
-                   kfc = controlledblock.interpolator
+                    kfc = controlledblock.interpolator
                 if kfc:
-                    self.import_keyframe_controller(kfc, b_armature_obj, bone_name, niBone_bind_scale, niBone_bind_rot_inv, niBone_bind_trans)
+                    self.import_keyframe_controller(kfc, b_armature_obj, bone_name, niBone_bind_scale,
+                                                    niBone_bind_rot_inv, niBone_bind_trans)
         # fallout: set global extrapolation mode here (older versions have extrapolation per controller)
         if kf_root.cycle_type:
             extend = self.nif_import.animationhelper.get_extend_from_cycle_type(kf_root.cycle_type)
             self.nif_import.animationhelper.set_extrapolation(extend, b_action.fcurves)
-        
-    def import_keyframe_controller(self, kfc, b_obj, bone_name=None, niBone_bind_scale=None, niBone_bind_rot_inv=None, niBone_bind_trans=None):
+
+    # TODO [animation] Is scale param required or can be removed, not used
```

Comment:
I think it will be needed if we ever support rest bone scale properly. Maybe, maybe not. Depends on the implementation: using pose mode scale & storing that in a dummy anim would not need it. I think right now rest bone scale is effectively broken, but I can't remember it ever working properly. Bone scale anims work fine.

Reply:
We can leave it as a todo anyways and revisit at a later point.

Reply:
Yup!